01.w_Defects/livelock.c

```diff
@@ -19,7 +19,7 @@
 pthread_mutex_t livelock_001_glb_A;
 pthread_mutex_t livelock_001_glb_B;
 
-int x,y;
+static int x,y;
```

Comment:
int x conflicts with another external declaration in race_condition.c so I made it static.